packages/video_player/video_player_avfoundation/lib/src/avfoundation_video_player.dart

```diff
@@ -139,14 +141,45 @@ class AVFoundationVideoPlayer extends VideoPlayerPlatform {
     });
   }
 
+  @override
+  Future<void> setMixWithOthers(bool mixWithOthers) {
+    return _api.setMixWithOthers(mixWithOthers);
+  }
+
   @override
   Widget buildView(int textureId) {
-    return Texture(textureId: textureId);
+    return buildViewWithOptions(
+      VideoViewOptions(
+        playerId: textureId,
+        // Texture view was the only supported view type before
+        // buildViewWithOptions was introduced. We pass it here to maintain
```

Comment:
I strongly prefer we not put "we" in code comments since it's often ambiguous who "we" is. In this case, I think the second sentence can just be removed entirely.